docs/getting-started.md

```diff
@@ -23,14 +23,25 @@ This document contains the following sections:
 
 ## Overview
 
-Luigi is an open source JavaScript framework for **micro frontends**. Micro frontends can be used to divide big frontend monoliths into smaller, simpler, independent chunks.
+Luigi is an open source JavaScript framework for **micro frontends**. Micro frontend architecture breaks down big frontend monoliths into smaller and simpler chunks which can be developed by independent teams.
 
-<!-- add-attribute:class:success -->
->**TIP:** Go to our [FAQ section](faq.md) to find more detailed answers to questions like *What are micro frontends?*
+The Luigi framework provides configuration options, API functions, and out-of-the-box features which make migrating to a micro frontend architecture easier. Luigi provides a consistent user navigation for all your micro frontends, ensuring a better user experience. 
 
-The Luigi framework provides configuration options, API functions, and out-of-the-box features which make migrating to a micro frontend architecture easier.
+Most importantly, Luigi is **technology-agnostic**, which means you can use virtually any toolkit (including, but not limited to: Angular, React, Vue, or UI5) to create your frontend. 
+
+### Benefits
+
+Some benefits of a micro frontend architecture include:
+
+| Micro frontends   |      Monolithic applications      | 
```

Comment:
i think some of the points here are a bit difficult, because they don't match exactly (e.g. independent teams vs. design team). Also i think a monolith is actually easier to test ;) Could we maybe simply remove the monolith column?

Reply:
Sure - in that case there is actually no need for a table.